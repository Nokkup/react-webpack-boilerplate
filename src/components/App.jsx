import React from "react";
import styles from "./App.module.scss";

function App() {
    return (
        <div className={styles.container}>
            <p className={styles.container__text}>Hello World!</p>
        </div>
    );
}

export default App;
