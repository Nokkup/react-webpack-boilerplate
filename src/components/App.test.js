import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

describe('App', () => {
  it('should render my component', () => {
    const component = shallow(<App />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
