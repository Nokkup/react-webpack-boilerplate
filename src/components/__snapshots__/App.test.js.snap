// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render my component 1`] = `
Array [
  <div
    className="container"
  >
    <p
      className="container__text"
    >
      Hello World!
    </p>
  </div>,
]
`;
